cvat/apps/webhooks/migrations/0005_auto_20230323_1253.py

```diff
@@ -16,11 +16,11 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='webhook',
             name='organization',
-            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='+', to='organizations.organization'),
+            field=models.ForeignKey(db_constraint=False, null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='+', to='organizations.organization'),
         ),
         migrations.AlterField(
             model_name='webhook',
             name='project',
-            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='+', to='engine.project'),
+            field=models.ForeignKey(db_constraint=False, null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='+', to='engine.project'),
```

Comment:
Why do you need db_constraint=False?

Reply:
It needs to prevent error when we delete project or organization and want to keep webhook.